docs/overview/deep_dive/data_types.rst

```diff
@@ -474,6 +474,68 @@ However, torch does not support ``uint32``, and so we cannot fully adhere to the
 Rather than breaking this rule and returning arrays of type ``uint8`` only with a torch backend, we instead opt to remove official support entirely for this combination of data type, function, and backend framework.
 This will avoid all of the potential confusion that could arise if we were to have inconsistent and unexpected outputs when using officially supported data types in Ivy.
 
+Another important point to note is that for cases where an entire dtype series is not supported or supported. For example if `float16`, `float32` and `float64` are not supported or is supported by a framework which could be a backend or frontend framework,
+then we simply identify that by simply replacing the different float dtypes with the str `float`. The same logic is applied to other dtypes such as `complex`, where we simply replace the entire dtypes with the str `complex`
+
+An example is :func:`ivy.fmin` with a tensorflow backend:
+
+.. code-block:: python
+
+    @with_supported_dtypes({"2.13.0 and below": ("float",)}, backend_version)
+    def fmin(
+        x1: Union[tf.Tensor, tf.Variable],
+        x2: Union[tf.Tensor, tf.Variable],
+        /,
+        *,
+        out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+    ) -> Union[tf.Tensor, tf.Variable]:
+        x1, x2 = promote_types_of_inputs(x1, x2)
+        x1 = tf.where(tf.math.is_nan(x1), x2, x1)
+        x2 = tf.where(tf.math.is_nan(x2), x1, x2)
+        ret = tf.experimental.numpy.minimum(x1, x2)
+        return ret
+
+As seen in the above code, we simply use the str `float` instead of writing all the float dtypes that are supported
+
+Another example is :func:`ivy.floor_divide` with a tensorflow backend:
+
+.. code-block:: python
+
+    @with_unsupported_dtypes({"2.13.0 and below": ("complex",)}, backend_version)
+    def floor_divide(
+        x1: Union[float, tf.Tensor, tf.Variable],
+        x2: Union[float, tf.Tensor, tf.Variable],
+        /,
+        *,
+        out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+    ) -> Union[tf.Tensor, tf.Variable]:
+        x1, x2 = ivy.promote_types_of_inputs(x1, x2)
+        return tf.experimental.numpy.floor_divide(x1, x2)
+
+As seen in the above code, we simply use the str `complex` instead of writing all the complex dtypes that are not supported
+
+Supported and Unsupported Data Types Attributes
+------------------------------------
```

Comment:
it would be great to have the number of `-` matching with the number of characters in the header as that's what's been done in other parts of the docs